index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <meta 
+  name="Description"
+  content="First project creating the first two sections of my Portfolio">
+  <title>Abdel-Khafid's Portfolio</title>
+</head>
+<body>
+  <header>
+    <div class="nav-container">
+        <div class="nav-logo">
+            <img class="my-lo" src="./assets/Abdel khafid SALAOU.png" alt="My_logo">
+            <div class="overlay">Hello new friend</div>
+        </div>
+        <nav class="topnav">
+            <div class="hamburer-lines">
+                <span class="line1"></span>
+                <span class="line2"></span>
+                <span class="line3"></span>
+            </div>
+            <a href="#news">Portfolio</a>            
+            <a href="#news">About</a>
+            <a href="#contact">Contact</a>
+            <a href="#">-- <br> --</a>
+        </nav>
+     </div>
+        
+    </div>
+</header>
+<main>
+    <section class="first-section">
+        <div class="major-container">
+            <div class="minor-container">
+                <div class="section-head">
+                    <h1>I'm Abdel-Khafid SALAOU</h1>
+                    <p>
+                        I am a Software developer! I can help you build a product,
+                        feature or website. Look through some of my works and
+                        experience! if you like what you see and have a project you
+                        need coded don't hesitate to contact me.
+                    </p>
+                </div>
+                <div class="connect">
+                    <p>LET'S CONNECT</p>
```

Comment:
Kindly change the `<p> ` tag to an `<a>` tag as suggested by the previous reviewer. It should be a link that will link to your contact page once clicked. 😞 ☺️

Reply:
Sorry for that I was so in a rush to complete the project and start today's one.